benefits/eligibility/templates/eligibility/includes/modal--senior-start-help.html

```diff
@@ -3,18 +3,31 @@
 {% load static %}
 
 {% block modal-content %}
-    <h2 class="me-4 me-md-0">
-        {% translate "eligibility.pages.start.senior.help.headline" %}
-        <img class="icon" width="197" height="26" src="{% static "img/login-gov-logo.svg" %}" alt="{% translate "core.logos.login_gov" context "image alt text" %}" />
-    </h2>
-    <div class="row">
-        <p class="pt-3">{% translate "eligibility.pages.start.senior.help.p[0]" %}</p>
-        <p class="pt-4">{% translate "eligibility.pages.start.senior.help.p[1]" %}</p>
-        <p class="pt-4">{% translate "eligibility.pages.start.senior.help.p[2]" %}</p>
-        <p class="pt-4">{% translate "eligibility.pages.start.senior.help.p[3]" %}</p>
-        <p class="pt-4">{% translate "eligibility.pages.start.senior.help.p[4]" %}</p>
-        <p class="pt-4 d-block d-sm-none">
-            <a href="#" data-bs-dismiss="modal" aria-label="Close">{% translate "core.modals.back" %}</a>
-        </p>
-    </div>
+  <h2 class="me-4 me-md-0">
+    {% translate "Verify your identity with " %}
+    <img class="icon" width="197" height="26" src="{% static "img/login-gov-logo.svg" %}" alt="{% translate "TODO" context "image alt text" %}" />
```

Comment:
Should the TODO here have something? Or it's an actual TODO?

Reply:
Yeah this is what it was in the PO file?

Reply:
ooo...! Can't it just be... "Login.gov"?

Reply:
[30268d0](https://github.com/cal-itp/benefits/pull/1626/commits/30268d0df45b7b390e837d7ef37c67a0f5d2c2ab)

Reply:
fix confirmed